index.html

```diff
@@ -475,6 +475,12 @@ <h2>Credentials</h2>
 cards, digital proofs of age, and digital educational certificates.
       </p>
 
+      <p class="note">
+It is possible to have a <a>credential</a>, such as a marriage certificate,
+that contains multiple <a>claims</a> about different <a>subjects</a> that are
```

Comment:
Would they be multiple claims or a single claim with statements about different subjects? Where a "claim" is an isolated graph.